### ISSUE TITLE
Lack of Monorepo Support

### ISSUE THREAD

**Issue by phamhongphuc** [Pilaton/vscode-npm-lens-support#2]
This extension has proven to be an invaluable tool for managing npm dependencies within my projects.
However, I recently encountered a limitation regarding its compatibility with monorepo setups. Specifically, the extension seems to only scan the package.json file located in the outermost directory of the project, thereby missing dependencies and scripts defined in packages nested within the monorepo structure.

As more developers are adopting monorepo architectures for managing large-scale projects, ensuring compatibility with such setups becomes increasingly crucial. Therefore, I wanted to bring this limitation to your attention and inquire whether there are any plans to address it in future updates of the extension.

It would greatly enhance the utility of Npm Lens for developers working with monorepos if the extension could recursively scan all `package.json` files within the project directory and its subdirectories.

Thank you for your attention to this matter. I'm looking forward to your response and any potential updates to improve the functionality of the Npm Lens extension.

**Comment by Pilaton** [comment]
The implementation of functionality for working with monorepos is already in progress.

**Comment by sanderkooger** [comment]
Yes! That's what I like to hear. I'll drop a comment, and thank you for your work, sir. Highly anticipated.... I await
